### PR TITLE
chore: fix lock workflow after update

### PR DESCRIPTION
**Pull request by armano2**
## PR Checklist

-   [x] Addresses an existing open issue: fixes #000
-   [x] That issue was marked as [accepting prs](https://github.com/typescript-eslint/typescript-eslint/issues?q=is%3Aopen+is%3Aissue+label%3A%22accepting+prs%22)
-   [x] Steps in [CONTRIBUTING.md](https://github.com/typescript-eslint/typescript-eslint/blob/main/CONTRIBUTING.md) were taken

## Overview

change missed while upgrading package, #4643, 
see https://github.com/dessant/lock-threads/blob/master/CHANGELOG.md#300-2021-09-27
https://github.com/typescript-eslint/typescript-eslint/actions/runs/2364718121

### CODE REVIEW

File: .github/workflows/lock.yml [hunk 1]
```diff
@@ -11,9 +11,9 @@ jobs:
       - uses: dessant/lock-threads@v3
         with:
           github-token: ${{ github.token }}
```
Comment (Josh-Cena) [inline .github/workflows/lock.yml:13]: This can be removed as well

Comment (armano2) [reply .github/workflows/lock.yml:13]: true, according to doc, all what we need is `issue-inactive-days` and `pr-inactive-days` as for rest of inputs we are using defaults, tho i'm not sure if we want to clean this up

**Review by bradzacher — APPROVED** [review]
thanks for fixing this!
I was wondering why some issues hadn't been locked in a while and forgot to look into it!

File: .github/workflows/lock.yml (hunk 1 above)
Comment (bradzacher) [reply .github/workflows/lock.yml:13]: i'm happy to leave it. no harm in having it!

**Pull request merged**